src/pages/choice-screen.html

```diff
@@ -0,0 +1,180 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Search Engine Choice</title>
+    <meta charset="UTF-8" />
+    <script src="common.js"></script>
+    <style>
+      @font-face {
+        font-family: Roboto;
+        src: url('../fonts/Roboto-Regular.ttf');
+        font-weight: 400;
+      }
```

Comment:
Can you split CSS into a separate file?

Reply:
Will do that when implementing next page